Server/app.js

```diff
@@ -0,0 +1,46 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+
+app.use(cors());
+
+// Links data arrays to a constant
+const resources = require("../Data/data.json")
+// Uses JSON Data File
+
+const results = require("../Data/data.js")
+// Uses JS Data File
+
+// Port Number
+const port = 8080;
+
+//Routes - First Route
+app.get('/', (req, res) => res.send('Hello World'))
+
+app.get('/topics', (req, res) => {
+    res.json(resources)
+});
+// Route using data.json
+
+app.get("/topics/:id", (req, res) => {
```

Comment:
I was confused by the use of :id . I thought it would be a classic numbered id. Took me a while to figure out I had to enter meat/vegetables !

Reply:
Because you guys used `:id` in `app.get`, we thought we would need to search localhost:8080/topics/<a number>. You guys need to change :id to something more descriptive so we know to do localhost:8080/topics/meat or localhost:8080/topics/vegetable